src/main/java/org/jenkinsci/plugins/pipeline/milestone/MilestoneStep.java

```diff
@@ -130,10 +148,9 @@ public void save() {
             LOGGER.log(Level.FINE, "save: {0}", milestonesByOrdinalByJob);
         }
 
+        @Deprecated
         public Map<String, Map<Integer, Milestone>> getMilestonesByOrdinalByJob() {
```

Comment:
We could probably clean up further here, like deleting the whole XML file. Maybe later.